### PR TITLE
feat: release v2

### PR DESCRIPTION
**Pull request by yyichenn**
*Issue #, if available:*
[495](https://github.com/aws-actions/amazon-ecr-login/issues/495)

*Description of changes:*
* BREAKING CHANGE: Change the default value of the `mask-password` input from `false` to `true`.
* Update README.md to reflect the new major version and the new `mask-password` default value. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -148,9 +137,6 @@ for details on how to configure ECR policies
 
 Use the action to output your Docker credentials for logging into ECR Private, then use the credentials to run your private image as a service in another job.
 
-> [!WARNING]
```
Comment (arjraman) [inline README.md:151]: Can you change the example for `Run an image as a service` to set `mask-password` to false, and keep this warning? This example won't work without it being set to false.

File: index.js [hunk 2]
```diff
@@ -106,7 +106,7 @@ function replaceSpecialCharacters(registryUri) {
 async function run() {
   // Get inputs
   const httpProxy = core.getInput(INPUTS.httpProxy, { required: false });
-  const maskPassword = core.getInput(INPUTS.maskPassword, { required: false }).toLowerCase() === 'true';
+  const maskPassword = (core.getInput(INPUTS.maskPassword, { required: false }).toLowerCase() || 'true') === 'true';
```
Comment (arjraman) [inline index.js:109]: Can we change this logic to only be `false` if maskPassword is explicitly set to `false`. Currently, if the customer writes something like `mask-password: asdf` by accident, the `mask-password` input becomes `false`.

File: README.md (hunk 1 above)
Comment (yyichenn) [reply README.md:151]: Added `mask-password` to false and updated warning.

File: index.js (hunk 2 above)
Comment (yyichenn) [reply index.js]: Updated.

**Review by arjraman — APPROVED** [review]

**Pull request merged**